tests/python2_vs_python3_sanity.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```

Comment:
I suppose the (most obvious) alternative to this lovely hack is calling out to a python interpreter, so I guess it can be excused. :)